zitcha_sentry_python_gcp_init/sentry_init.py

```
@@ -0,0 +1,21 @@
+import os
+import urllib.request
```

Ruff (0.8.2)

`urllib.request` imported but unused

Remove unused import: `urllib.request`

(F401)